src/app/query-builder/query-builder.sample.html

```diff
@@ -1,17 +1,52 @@
-<div class="sample-content">
-    <div class="sample-column">
-        <size-selector #sizeSelector></size-selector>
+<div class="qb-sample-content">
+    <div class="qb-sample-column">
+        <button igxButton="contained" (click)="qbDialog.open()">Open QB inside a dialog</button>
+
+        <igx-dialog #qbDialog closeOnOutsideSelect>
+            <igx-query-builder [entities]="this.entities">
+                <igx-query-builder-tree ></igx-query-builder-tree>
```

Comment:
Is this `<igx-query-builder-tree>` really meant to be here?

Reply:
It shouldn't be here